torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1403,7 +1403,7 @@ XLATensor XLATensor::full_like(const XLATensor& input,
 }
 
 XLATensor XLATensor::gather(const XLATensor& input, int64_t dim,
-                            const XLATensor& index) {
+                            const XLATensor& index, bool sparse_grad) {
```

Comment:
Don't you need to pass `sparse_grad` to line `1418`?

Reply:
```
return input.CreateFrom(torch::lazy::MakeNode<Gather>(
      input.GetIrValue(), canonical_dim, index.GetIrValue(), sparse_grad));

```

Reply:
It's being passed already?

Reply:
Actually, it's not picked in this diff?! My local branch is up to date and shows the correct one -- but the github file view doesn't.

Reply:
Ok, VS CODE didn't write out the change to the file, resyncing and recommiting. Thanks @miladm